package.json

```diff
@@ -23,6 +23,7 @@
     "image-size": "^1.1.1",
     "jszip": "^3.10.1",
     "localforage": "^1.10.0",
+    "toastify-js": "^1.12.0",
```

Comment:
Our company prefer using exact versions. Can you do `--save-exact`?

Reply:
Also when you move CDN, can you help clean up all packages to use exact version?